src/split-panel/implementation.tsx

```diff
@@ -116,7 +116,7 @@ export const SplitPanelImplementation = React.forwardRef<HTMLElement, SplitPanel
               ariaLabel={i18nStrings.closeButtonAriaLabel}
               ariaExpanded={isOpen}
             />
-          ) : position === 'side' ? null : (
+          ) : isToolbar || position === 'side' ? null : (
```

Comment:
this open button was hijacking the focus for a brief second and the throwing a false positive to not suppres the tooltip